README.md

```diff
@@ -4,63 +4,27 @@
 
 ## Use case
 
-A js frontend uses a backend api which, in turn,  connects to keycloak for user authentication. The frontend tries to connect to a backend url and redirects the browser to a keycloak url if the first connection returns an error (because the user isn't authenticated).
+A Frontend Javascript service that integrates with the [Keycloak authentication service](http://keycloak.jboss.org/), guiding users to a login portal when a 401 Unauthorised response is received from an API.
 
 ## Install
 
 ```bash
 $ npm i keycloak-redirect
 ```
 
-## Usage of the library
+## Configuration
 
-Require the package, then use the object that was exposed using Browserify to return the function `authenticate`, passing in all mandatory arguments: a config object, a XMLHttpRequest object, and window.
+Import the package, then use the object exposed to return the function `authenticate`, passing in all mandatory arguments: a config object, a XMLHttpRequest object, and window.
 
 ```js
     
-   const keycloakRedirect = require('keycloak-redirect');
+   import keycloakRedirect from 'keycloak-redirect';
    
    var config = {
        backend: "http://yourBackendUrl.com",
        clientId: "yourClientId",
        keycloakUrl: "http://yourKeycloakUrl.com"
    };
    
-   var client = new XMLHttpRequest();
-   
-   keycloakRedirect.authenticate(config, client, window);
-```
-
-the `backend`, `clientId` and `keycloakUrl` keys on the config object are mandatory.
-
-## Run the build scripts
```

Comment:
sure, but the `npm test` is probably a still valid line?
along with something about the new `npm compile`